docs/dev/new_check_howto.md

```diff
@@ -23,7 +23,7 @@ Python 2.7 needs to be available on your system. It is strongly recommended to c
 
 You'll also need `docker-compose` in order to run the test harness. 
 
-## Setup
+## Integration Setup
```

Comment:
nit: the setup is relative to the environment, in order to provide all the requirements. After this change I read it as the "setup of the integration" (which technically doesn't exist yet at this stage).